bin/sozo/src/commands/build.rs

```diff
@@ -57,19 +69,50 @@ impl BuildArgs {
     }
 }
 
+pub fn print_stats(contract_statistic: ContractStatistics, config: &Config) {
+    let contract_name = contract_statistic.contract_name;
+    let number_felts = contract_statistic.number_felts;
+    let file_size = contract_statistic.file_size;
+    config.ui().print(format!("---------------Contract Stats for {contract_name}---------------"));
+    config.ui().print(format!(
+        "- Contract bytecode size (Number of felts in the program): {number_felts}\n"
+    ));
+    config.ui().print(format!("- Contract Class size: {file_size} bytes\n"));
```

Comment:
```suggestion
    config.ui().print(format!("---------------Contract Stats for {contract_name}---------------"));
    config.ui().print(format!(
        "- Contract bytecode size (Number of felts in the program): {number_felts}\n"
    ));
    config.ui().print(format!("- Contract Class size: {file_size} bytes\n"));
```

let's not use the `-`'s as it would it would not lined up and appears very inconsistent